src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,27 @@
+$card-width: 200px;
+@function catalogSize() {
+  @return $card-width * 5;
+}
```

Comment:
Why catalog size `$card-width * 5`?

Reply:
I fixed it and made it more understandable.